test/Fail/Issue5448-1.err

```diff
@@ -1,6 +1,10 @@
-Issue5448-1.agda:8,1-19
-Agda.Primitive.Cubical.primTransp (λ i → P (x i)) φ
-(subst P refl
- (Agda.Primitive.Cubical.primTransp (λ i → P x₁) φ
-  x₂)) is not usable at the required modality .
-when checking the definition of subst
+Issue5448-1.agda:8,1-15
+This clause has target type P x which is not usable at the required
+modality
```

Comment:
I think it also makes sense to change how the default modality is printed, as you pointed out somewhere else.